packages/registry/docs/conceptual-guides/add-on-loader.md

```diff
@@ -0,0 +1,97 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:title": "Add-ons configuration loader"
+    "keywords": "@plone/registry, registry, add-ons, loader"
+---
+
+# Add-ons configuration loader
+
+The `@plone/registry` compatible add-ons are able to load configuration into the configuration registry.
+This is achieved by using the `main` key entry point specified in the add-on {file}`package.json`.
+This should be a JavaScript or TypeScript file, eg. {file}`index.ts` or {file}`index.js` placed somewhere (generally in the root) of your add-on.
+
+```json
+{
+  "main": "index.ts",
+}
+```
+
+This file should contain a default export with a function with this signature:
+
+```ts
+import type { ConfigType } from '@plone/registry';
+
+export default function loadConfig(config: ConfigType) {
+  // You can mutate the configuration object in here
+  return config;
+}
+```
+
+`@plone/registry` has a helper utility `createAddonsLoader` for generating an add-ons loader file that contains the code needed to load the add-ons configuration of all the registered add-ons, keeping the order in which they were defined.
```

Comment:
Let's try to keep sentences short. When it gets too long or runs on, it's probably better to break it into multiple shorter sentences.

```suggestion
`@plone/registry` has a helper utility `createAddonsLoader` which generates an add-ons loader file.
That file contains the code needed to load the add-ons configuration of all the registered add-ons, keeping the order in which they were defined.
```

Reply:
" which generates an add-ons loader file." -> " which generates **the** add-ons loader file."
I think this is better, the loader file is unique. What do you think?